extensions/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/NoopHostnameVerifier.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.restclient.runtime;
```

Comment:
If it's exposed to the user and not some implementation detail, it should be at the root, not in the runtime package.